src/main/java/edu/gatech/cs2340/trydent/internal/JavaFXManager.java

```diff
@@ -0,0 +1,31 @@
+package edu.gatech.cs2340.trydent.internal;
+
+import javafx.scene.Group;
+import javafx.scene.Scene;
+
+/**
+ * Class to manage starting/stopping JavaFX.
+ *
+ * @author Garrett Malmquist
+ */
+public interface JavaFXManager {
```

Comment:
Actually it needs to be public because it's in a different package than TrydentEngine, which uses it.